src/Component/src/Exception/UnexpectedTypeException.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Resource\Exception;
+
+class UnexpectedTypeException extends \InvalidArgumentException
```

Comment:
Please add `Sylius\Resource\Exception\InvalidArgumentException extends \InvalidArgumentException implements ExceptionInterface`.

```suggestion
class UnexpectedTypeException extends InvalidArgumentException
```

Reply:
Same for all other exceptions